spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -669,7 +677,7 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     val r = new Random(0)
     val values = Seq(
       Float.MaxValue,
-      Float.MinPositiveValue,
+      // Float.MinPositiveValue,
       Float.MinValue,
```

Comment:
MinPositiveValue is so far the only value that makes trouble for both double/float

Reply:
Rather than comment it out in the shared `generateFloats` method, I think it would be better to just have a separate list of values for testing this specific cast.

Reply:
yes right will do

Reply:
done